change/@ni-nimble-components-c9f2f270-d0d3-478a-b43a-c0777af5fc44.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

In #830 I'm proposing that all changes to experimental/in-work components can be marked as `patch` to reduce churn on clients. Unless someone has a reason to disagree, I'd propose we start doing that even in this initial PR.

```suggestion
  "type": "patch",
```